contrib/startup_regtest.sh

```diff
@@ -53,10 +53,14 @@ fi
 
 if [ -z "$PATH_TO_BITCOIN" ]; then
 	if [ -d "$HOME/.bitcoin" ]; then
-		PATH_TO_BITCOIN="$HOME/.bitcoin"
+	    PATH_TO_BITCOIN="$HOME/.bitcoin"
 	else
+	    if [ -d "$HOME/Library/Application Support/Bitcoin/" ]; then
```

Comment:
Maybe better avoid an extra level of indentation using `elif` instead of `else if`.